app/code/community/Fballiano/ImageCleaner/Block/Adminhtml/Fbimagecleaner/Renderer/Image.php

```diff
@@ -18,16 +18,16 @@ public function render(Varien_Object $row)
         $max_width = Mage::getStoreConfig('admin/fb_image_cleaner/thumbnail_max_width');
 
         switch ($row->getEntityTypeId()) {
-            case 3:
+            case Mage::getModel('catalog/category')->getResource()->getTypeId():
                 $url .= "catalog/category/";
                 break;
-            case 4:
+            case Mage::getModel('catalog/product')->getResource()->getTypeId():
                 $url .= "catalog/product/";
                 break;
-            case -3:
+            case -Mage::getModel('catalog/category')->getResource()->getTypeId():
```

Comment:
genuine question, does this syntax work?

Reply:
it does for PHP 7.4.x, 8.2.x
It is like doing:
```php
$foo = 5;
$bar = -$foo;
echo $bar;
```